tests/unit/module_utils/test_operations.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2023, Denys Denysyev (ddenysyev@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING.GPLv3 or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.purestorage.fusion.plugins.module_utils.errors import (
+    OperationException,
+)
+from ansible_collections.purestorage.fusion.tests.unit.mocks.exceptions_mock import (
+    ApiExceptionsMock,
+)
+from ansible_collections.purestorage.fusion.tests.unit.mocks.operation_mock import (
+    OperationMock,
+    OperationStatus,
+)
+
+__metaclass__ = type
+
+try:
```

Comment:
These try/catch blocks should probably be left out in tests and mocks since their purpose is to suppress immediate errors in Ansible to be able to return them better formatted later in main. In tests we instead want to fail as soon and loud as possible and the tests should not affect the modules themselves, if I am not mistaken.